setup.cfg

```diff
@@ -11,7 +11,6 @@ source = zib_uploader
 
 [tool:pytest]
 testpaths = tests
-addopts = --cov --cov-report xml --cov-report term --cov-report html
```

Comment:
I removed this because it prevented me to run any tests. Not sure why we need it?

Reply:
I believe it's from the escience cookiecutter template but it's fine to remove it. For local coverage analysis I like to use pycharms built-in coverage anyway. I noticed that having coverage installed locally prevents me from using the debugger.

Reply:
Ok, cool